From 1a5279d9d2602d2696ed471a4af64caa51348b66 Mon Sep 17 00:00:00 2001
From: Dylan Yip <dylan.yip@xilinx.com>
Date: Tue, 9 Mar 2021 12:36:49 -0800
Subject: [PATCH 1776/1851] arm64: zynqmp: Label DP layers by default

commit 94849c61ff2774fdf89f22d8c339a74932c37573 from
https://github.com/Xilinx/linux-xlnx.git

Label DisplayPort video and graphic layers so they can be used to pass
live input to. This is useful when using a xlnx bridge to provide live
video data to the DisplayPort subsystem.

Signed-off-by: Dylan Yip <dylan.yip@xilinx.com>
Signed-off-by: Jianqiang Chen <jianqiang.chen@xilinx.com>
Acked-by: Varunkumar Allagadapa <varunkumar.allagadapa@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 arch/arm64/boot/dts/xilinx/zynqmp.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/xilinx/zynqmp.dtsi b/arch/arm64/boot/dts/xilinx/zynqmp.dtsi
index b0b306ed796d..b7a91769fb2c 100644
--- a/arch/arm64/boot/dts/xilinx/zynqmp.dtsi
+++ b/arch/arm64/boot/dts/xilinx/zynqmp.dtsi
@@ -1155,14 +1155,14 @@
 
 			power-domains = <&zynqmp_firmware PD_DP>;
 
-			vid-layer {
+			vid_layer: vid-layer {
 				dma-names = "vid0", "vid1", "vid2";
 				dmas = <&xlnx_dpdma 0>,
 				       <&xlnx_dpdma 1>,
 				       <&xlnx_dpdma 2>;
 			};
 
-			gfx-layer {
+			gfx_layer: gfx-layer {
 				dma-names = "gfx0";
 				dmas = <&xlnx_dpdma 3>;
 			};
-- 
2.31.1

